src/pages/Locations/LocationDetails/components/LocationPhotosGrid/LocationPhotosGrid.jsx

```diff
@@ -0,0 +1,197 @@
+                            <div style={{display: 'flex', justifyContent: 'center'}}>
+                                <img width={1080} height={900} className="mt-10 main-photo w-4_10 mr-5 start-rounded"
+                                     alt="location"
+                                     src={props.photos[0]}/>
+                                <img width={1080} height={900} style={{width: "40%"}}
+                                     className="mt-10 lg-visible end-rounded" alt="location"
+                                     src={props.photos[1]}/>
+                            </div>
+                        </div>
+                        : null}
+                    {props.photos.length === 3 ?
+                        <div style={{display: 'flex', justifyContent: 'center'}}>
+                            <div className="w-6_10 lg-w-1_2 md-w-1 mr-5">
+                                <img width={1080} height={600} className="mt-10 start-rounded md-end-rounded"
+                                     alt="location"
+                                     src={props.photos[0]}/>
+                            </div>
+                            <div className="alternative-photos lg-w-1_2 w-3_10 md-visible">
+                                <img width={1080} height={600} className="mt-10 lg-visible top-end-rounded"
+                                     alt="location"
+                                     src={props.photos[1]}/>
+                                <img width={1080} height={600} className="mt-10 bottom-end-rounded lg-top-end-rounded"
+                                     alt="location"
+                                     src={props.photos[2]}/>
+                            </div>
+                        </div>
+                        : null}
+                    {props.photos.length === 4 ?
+                        <div style={{display: 'flex', justifyContent: 'center'}}>
+                            <div className="w-6_10 lg-w-1_2 md-w-1 mr-5">
+                                <img width={1080} height={700} className="mt-10 start-rounded md-end-rounded"
+                                     alt="location"
+                                     src={props.photos[0]}/>
+                            </div>
+                            <div className="alternative-photos lg-w-1_2 w-1_6 md-visible">
+                                <img width={1080} height={700} className="mt-10 lg-visible top-end-rounded"
+                                     alt="location"
+                                     src={props.photos[1]}/>
+                                <img width={1080} height={700} className="mt-10 lg-visible" alt="location"
+                                     src={props.photos[2]}/>
+                                <img width={1080} height={700} className="mt-10 bottom-end-rounded lg-top-end-rounded"
+                                     alt="location"
+                                     src={props.photos[3]}/>
```

Comment:
Codacy has a fix for the issue: [Replace `·····································src={props.photos[3]}/>` with `··················src={props.photos[3]}`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
                  src={props.photos[3]}
```